elastic/elastic-agent
# Fixingspacek8s templates

- Bug

## What does this PR do?

Adding newline and removing space from k8s templates that are auto-generated.
The m

## Why is it important?

The issue reported [here](https://buildkite.com/elastic/kibana-pull-request/builds/204766#018ef3b5-09a1-49f7-aa0b-8c3ab48fcdff)
Part of https://github.com/elastic/kibana/pull/181213#issuecomment-2070237977

## How to test this PR locally

1. Clone elastic-agent repo
2. cd elastic-agent/deploy/kubernetes
3.  WITHOUTCONFIG=true make generate-k8s
4. ./creator_k8s_manifest.sh .

The `elastic_agent_manifest.ts` should include the generated k8s template

## Related issues

- Relates #https://github.com/elastic/kibana/pull/181213
